<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QProgressBar" name="moduleInfo_progressbar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="moduleInfo_cancel_button">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="moduleInfo_installation_label">
     <property name="text">
      <string>No installation found</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="moduleInfo_stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="moduleInfo_stackedWidget_pageInstall">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>New installation</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="db_parameters_CreateAndGrantRoles_install_checkBox">
            <property name="text">
             <string>Create and grant roles</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="db_demoData_checkBox">
              <property name="text">
               <string>Install demo data</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="db_demoData_comboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="ParametersGroupBox" name="parameters_groupbox">
            <property name="title">
             <string>Parameters</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6"/>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moduleInfo_install_pushButton">
            <property name="text">
             <string>Install</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="moduleInfo_stackedWidget_pageUpgrade">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="5" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QPushButton" name="uninstall_button">
           <property name="text">
            <string>Uninstall</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="moduleInfo_upgrade_pushButton">
           <property name="text">
            <string>Upgrade</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="db_parameters_CreateAndGrantRoles_upgrade_checkBox">
           <property name="text">
            <string>Create and grant roles</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="moduleInfo_selected_label">
         <property name="text">
          <string>Selected module version</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="moduleInfo_stackedWidget_pageMaintain">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="5" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="2">
          <widget class="QPushButton" name="uninstall_button_maintain">
           <property name="text">
            <string>Uninstall</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="moduleInfo_drop_app_pushButton">
           <property name="text">
            <string>Drop app</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="moduleInfo_roles_pushButton">
           <property name="text">
            <string>Create and grant roles</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="moduleInfo_recreate_app_pushButton">
           <property name="text">
            <string>(Re)create app</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="moduleInfo_selected_label_maintain">
         <property name="text">
          <string>Selected module version</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParametersGroupBox</class>
   <extends>QGroupBox</extends>
   <header>oqtopus.gui.parameters_groupbox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
